### DISCUSSION TITLE
Compatibility between HW and GO sample sets?

### DISCUSSION THREAD

**Discussion by jason-barba** [GrandOrgue/grandorgue#882] [General]
I am sorry if this is answered elsewhere, but I cannot find the information.

How compatible are Hauptwerk sample sets with GrandOrgue, if at all? I have looked very quickly into a HW version 1 .organ file and it shows similarities with and GO ODF.

**Comment by rousseldenis** [comment]
@jason-barba Indeed, HW v1 had the same format as they have the same development origin.

But that story stops there as both forked in two relatively different softwares. As the principle remain the same based on rendering audio samples as near as possible from the original instrument, the definition files are very different (by nature, one text file, one xml).

Some are writting GO definition files for open HW ones if they are not provided by the sampler (e.g.: Piotr Grabowski at the beginning).

**Comment by larspalo** [comment]
As @rousseldenis said the sampleset definitions differ, so except HW1 samplesets you cannot load/open any HW samplesets in GO. However, if (and only if) the samples of a HW sampleset is unencrypted they can be played in GO provided an ODF is constructed. There are some available if you scan the resources.
